scss/_progress.scss

```diff
@@ -7,101 +7,69 @@
-  @include border-radius($border-radius);
+  @include border-radius($progress-border-radius);
   @include box-shadow(inset 0 .1rem .1rem rgba(0,0,0,.1));
+  background-color: $progress-bg;
 }
+
 .progress[value]::-webkit-progress-value {
-  background-color: $progress-bar-color;
-  border-top-left-radius: $border-radius;
-  border-bottom-left-radius: $border-radius;
+  border-bottom-left-radius: $progress-border-radius;
+  border-top-left-radius: $progress-border-radius;
 }
+
 .progress[value="100"]::-webkit-progress-value {
-  border-top-right-radius: $border-radius;
-  border-bottom-right-radius: $border-radius;
+  border-bottom-right-radius: $progress-border-radius;
+  border-top-right-radius: $progress-border-radius;
 }
 
-// Firefox styles must be entirely separate or it busts Webkit styles.
-//
-// Commented out for now because linter.
-//
-// $-moz-document url-prefix() {
-//   .progress[value] {
-//     background-color: $progress-bg;
-//     .border-radius($border-radius);
-//     .box-shadow(inset 0 .1rem .1rem rgba(0,0,0,.1));
-//   }
-//   .progress[value]::-moz-progress-bar {
-//     background-color: $progress-bar-color;
-//     border-top-left-radius: $border-radius;
-//     border-bottom-left-radius: $border-radius;
-//   }
-//   .progress[value="0"]::-moz-progress-bar {
-//     color: $gray-light;
-//     min-width: 2rem;
-//     background-color: transparent;
-//     background-image: none;
-//   }
-//   .progress[value="100"]::-moz-progress-bar {
-//     border-top-right-radius: $border-radius;
-//     border-bottom-right-radius: $border-radius;
-//   }
-// }
-
-// IE9 hacks to accompany custom markup. We don't need to scope this via media queries, but I feel better doing it anyway.
-@media screen and (min-width:0\0) {
-  .progress {
-    background-color: $progress-bg;
-    @include border-radius($border-radius);
-    @include box-shadow(inset 0 .1rem .1rem rgba(0,0,0,.1));
-  }
-  .progress-bar {
-    display: inline-block;
-    height: $spacer-y;
-    text-indent: -999rem; // Simulate hiding of value as in native `<progress>`
-    background-color: $progress-bar-color;
-    border-top-left-radius: $border-radius;
-    border-bottom-left-radius: $border-radius;
-  }
-  .progress[width^="0"] {
-    min-width: 2rem;
-    color: $gray-light;
-    background-color: transparent;
-    background-image: none;
-  }
-  .progress[width="100%"] {
-    border-top-right-radius: $border-radius;
-    border-bottom-right-radius: $border-radius;
-  }
+.progress-bar {
+  border-bottom-left-radius: $progress-border-radius;
+  border-top-left-radius: $progress-border-radius;
+  display: inline-block;
+  height: $spacer-y;
+  text-indent: -999rem; // Simulate hiding of value as in native `<progress>`
 }
 
+.progress[width^="0"] {
+  background-color: transparent;
+  background-image: none;
+  color: $gray-light;
+  min-width: 2rem;
+}
+.progress[width="100%"] {
+  border-bottom-right-radius: $progress-border-radius;
```

Comment:
Properties should be ordered border-top-right-radius, border-bottom-right-radius